analyses/malawi/notebooks/read_in_data.py

```diff
@@ -0,0 +1,184 @@
+import sys
+import os
+from pathlib import Path
+import geopandas as gpd
+from rasterstats import zonal_stats
+import rioxarray
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(__file__))).parents[2]}/"
+sys.path.append(path_mod)
+
+from src.indicators.drought import ecmwf_seasonal
+from src.malawi import get_ecmwf_seasonal_data as gesd
+from src.indicators.drought.config import Config
+
+#TODO: not sure which functions to include here and which in the ecmwf_seasonal file
```

Comment:
I originally made this file for Bangladesh when I had multiple notebooks using the same methods to avoid duplication. Now that we are using them for other countries, we should probably generalize them somewhere in `src` (although I'm not sure how much overlap there is between GloFAS and ECMWF methods here)

Reply:
Moved it to `src/indicators/drought/ecmwf_seasonal/processing.py`. Do you think that makes sense?